docs/release-notes/v0.7.0.rst

```diff
@@ -0,0 +1,254 @@
+  (PRs :pr:`1625`, :pr:`1644`)
+
+* ``{modifier}_builder`` classes are added for all modifiers.
+  (PRs :pr:`1625`)
+  For example, :class:`pyhf.modifiers.histosys.histosys_builder`.
+
+* When using ``pyhf.writexml`` and the ``normfactor`` parameter config is missing
+  ``inits`` or ``bounds``, fall back to using default values.
+  (PRs :pr:`1819`)
+
+* Supported options for :func:`pyhf.infer.hypotest` can now be passed as kwargs
+  through the :func:`pyhf.infer.intervals.upper_limits.upper_limit` API.
+  (PR :pr:`1613`)
+  This now enables things like using :class:`pyhf.infer.calculators.ToyCalculator`
+  as the calculator used for the hypothesis test scan:
+
+  .. code:: pycon
+
+      >>> import numpy as np
+      >>> import pyhf
+      >>> pyhf.set_backend("jax")
+      >>> model = pyhf.simplemodels.uncorrelated_background(
+      ...     signal=[12.0, 11.0], bkg=[50.0, 52.0], bkg_uncertainty=[3.0, 7.0]
+      ... )
+      >>> observations = [51, 48]
+      >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
+      >>> scan = np.linspace(0, 5, 21)
+      >>> obs_limit, exp_limits, (scan, results) = pyhf.infer.intervals.upper_limits.upper_limit(
+      ...     data, model, scan, return_results=True, calctype="toybased", ntoys=3000
+      ... )
+
+* Allow for fit parameter values from required fits in ``pyhf.infer.test_statistics``
+  functions to be returned by use of ``return_fitted_pars`` kwarg with the
+  ``pyhf.infer.test_statistics`` functions and ``return_calculator`` kwarg with
+  :func:`pyhf.infer.hypotest`.
+  (PR :pr:`1554`)
+
+* A ``validate`` kwarg has been added to :func:`pyhf.workspace.Workspace` and
+  :func:`pyhf.pdf.Model` to allow skipping validation.
+  (PR :pr:`1646`)
+  This should only be used by expert users who understand the risks.
+
+* A ``pyhf.default_backend`` has been added that is configurable through a
+  ``default`` kwarg in :func:`pyhf.set_backend`.
+  (PR :pr:`1646`)
+  This allows setting the ``pyhf.default_backend`` to be different from the value of
+  ``pyhf.tensorlib`` returned by :func:`pyhf.get_backend`, which can be useful in situations
+  where differentiable model construction is needed.
+
+  .. code:: pycon
+
+      >>> import jax
+      >>> import pyhf
+      >>> pyhf.set_backend("jax", default=True)
+      >>> pyhf.set_backend("numpy")
+      >>> pyhf.get_backend()
+      (<pyhf.tensor.numpy_backend.numpy_backend object at 0x...>, <pyhf.optimize.scipy_optimizer object at 0x...>)
+      >>> pyhf.default_backend
+      <pyhf.tensor.jax_backend.jax_backend object at 0x...>
+      >>> def example_op(x):
+      ...     return 2 * pyhf.default_backend.power(pyhf.default_backend.astensor(x), 3)
+      ...
+      >>> example_op([2.0])
+      DeviceArray([16.], dtype=float64)
+      >>> jax.jacrev(jax.jit(example_op))([2.0])
+      [DeviceArray([24.], dtype=float64, weak_type=True)]
+
+* :func:`pyhf.pdf.Model` now allows for both :class:`list` and ``pyhf.tensorlib`` objects
```

Comment:
this phrasing is confusing as it implies that this is a change to `pyhf.pdf.Model` (it's not, but a change to schema validation to allow tensor types - also needed for diffability)